pkg/sink/codec/simple/avro.go

```diff
@@ -265,98 +276,104 @@ func (a *avroMarshaller) newDMLMessageMap(
 
 		holder := genericMapPool.Get().(map[string]interface{})
 		holder["com.pingcap.simple.avro.Checksum"] = cc
-		m["checksum"] = holder
+		dmlMessagePayload["checksum"] = holder
 	}
 
 	if event.IsInsert() {
 		data := a.collectColumns(event.Columns, event.TableInfo, onlyHandleKey)
-		m["data"] = data
-		m["type"] = string(DMLTypeInsert)
+		dmlMessagePayload["data"] = data
```

Comment:
I would prefer use `m`, since it's usage range is quite small, no need to make the variable name too verbose.

Reply:
This var is frequently used in this function, and `m` make it hard to understand what it is.

Reply:
You can name it as `payload`, the `dmlMessagePayload` is not good name IMHO, it's too mouthful